tests/subsys/bootloader/boot_lock_kmu_keys/Kconfig.sysbuild

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2025 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config KMU_KEYS_LOCK_TEST_KEY_FILE
+	string "Path to the private key file used by the test"
+	default "$(ZEPHYR_NRF_MODULE_DIR)/tests/subsys/bootloader/boot_lock_kmu_keys/private_key.pem"
+
+config BOOT_SIGNATURE_KEY_FILE
+	default KMU_KEYS_LOCK_TEST_KEY_FILE
+
+config SECURE_BOOT_SIGNING_KEY_FILE
+	default KMU_KEYS_LOCK_TEST_KEY_FILE
+
```

Comment:
delete whole file, use `sysbuild.conf` to set files

Reply:
Yes, so remove it and reference the other Kconfigs in the script and have the others set in a `sysbuild.conf` file